elements/pl-external-grader-results/pl-external-grader-results.py

```diff
@@ -139,17 +139,20 @@ def render(element_html, data):
                             test["max_points"] = round_value(
                                 results_test.get("max_points")
                             )
-                            correct = test["max_points"] == test["points"]
 
-                            # Don't show points for test cases that are 0/0.
-                            # Note that we compare with a string instead of a
-                            # number because we converted it to a string with
-                            # the above call to `round_value()`.
-                            if correct and test["max_points"] == "0":
-                                test["show_points"] = False
+                            if test["max_points"] != test["points"]:
+                                test["results_color"] = "#F44336"
+                                test["results_icon"] = "fa-times"
+                            # Don't consider points for test cases that are 0/0
+                            # to be correct. We compare with a string because
+                            # `round_value()` returns a string.
+                            elif test["max_points"] == "0":
+                                test["results_color"] = "rgba(0, 0, 0, 0.3)"
```

Comment:
Does this meet the [WCAG color contrast requirements](https://webaim.org/resources/contrastchecker/)?

Reply:
It does not. I'll play a bit with some numbers. That said, the green color for correct doesn't pass it either (red passes only the WCAG AA test for large text, which given this is bold/16pt is ok here).

Reply:
Here's with CSS colors, crimson, darkgreen and dimgray. These all meet the standards.

<img width="198" alt="image" src="https://user-images.githubusercontent.com/1004907/231246191-37ad3efa-f6eb-4927-af4f-803b88a2672a.png">

Reply:
IMO the red/green there are borderline too dark to be distinguishable from each other, and the green appears almost black (caveat: my optometrist has said I am slightly colorblind with blues/greens). Are there lighter shades that still pass contrast requirements?

Reply:
<img width="146" alt="image" src="https://user-images.githubusercontent.com/1004907/231255249-99241b29-1a6a-4fc0-aeb7-b2f5a061a3fb.png">

These are E70000, 008900 and 757575 for red, green and grey, respectively. These still pass the requirements. Not yet pushed, let me know (or maybe we should get someone else's opinion, since I'm also colorblind 😬 )?

Reply:
When we revisit this in earnest (it that this issue?), we will want to move away from red/green contrast explicitly because it's the most common form of colorblindness.

Reply:
The good news is that we don't *solely* rely on color to convey pass/fail/info - we have icons and points as well.

@jonatanschroeder I think that looks improved - can you push that so I can play with it locally?

Reply:
It has been pushed in e728fd0.